proposals/lip-0047.md

```diff
@@ -506,92 +507,87 @@ The following two steps are executed as part of the genesis block processing, se
 
 To initialize the state of a genesis block `g`, the following logic is executed. If any step fails, the block is discarded and has no further effect.
 
-Let `genesisBlockAssetBytes` be the bytes included in the block assets for the PoA module and let `genesisBlockAssetObject` be the deserialization of `genesisBlockAssetBytes` according to the `genesisPoAStoreSchema` schema, given below, then: 
-
-* Checks for the `genesisBlockAssetObject.validatorSubstore`: 
-  * Check that the `address` property of every entry `validator` in the `genesisBlockAssetObject.validatorSubstore.validators` array is distinct pairwise.
-  * Check that the `name` property of every entry `validator` in the `genesisBlockAssetObject.validatorSubstore.validators` array contains only characters from the set `[a-z0-9!@$&_.]`, is not empty and is at most `MAX_LENGTH_NAME` characters long.
-  * Check that the `generatorKey` property of every entry `validator` in the `genesisBlockAssetObject.validatorSubstore.validators` array has length 32.
-  * Check that the `blsKey` property of every entry `validator` in the `genesisBlockAssetObject.validatorSubstore.validators` array has length 48.
-  * Check that the `proofOfPossession` property of every entry `validator` in the `genesisBlockAssetObject.validatorSubstore.validators` array has length 96.
-
-* Checks for the `genesisBlockAssetObject.snapshotSubstore`: 
-  * The entries in the `genesisBlockAssetObject.snapshotSubstore.activeValidators` array are ordered lexicographically according to `address`.
-  * For every element `activeValidator` in the `genesisBlockAssetObject.snapshotSubstore.activeValidators` array, there is an entry with `address == activeValidator.address` in the `genesisBlockAssetObject.validatorSubstore.validators` array.
-  * The `weight` property of every element `activeValidator` in the `genesisBlockAssetObject.snapshotSubstore.activeValidators` array is a positive integer.
-  * Let `totalWeight` be the sum of the `weight` property of every element `activeValidator`. Then `totalWeight` has to be less than or equal to `MAX_UINT64`.
-  * The value of `genesisBlockAssetObject.snapshotSubstore.threshold` is within the following range:
+Let `genesisBlockAssetBytes` be the bytes included in the `data` property of a deserialized entry in the block assets for the PoA module and let `assetPoA` be the deserialization of `genesisBlockAssetBytes` according to the `genesisPoAStoreSchema` schema, then: 
```

Comment:
```suggestion
Let `genesisBlockAssetBytes` be the bytes included in the `data` property of a deserialized entry in the block assets for the PoA module and let `assetPoA` be the deserialization of `genesisBlockAssetBytes` according to the `genesisPoAStoreSchema` schema. Then: 
```

Reply:
Just a suggestion as the sentence is very long.

Reply:
updated